locations/spiders/gwk_travelex.py

```diff
@@ -17,26 +18,22 @@ def start_requests(self):
 
     def parse(self, response):
         data = response.json()
-
         item_categories = data.get("items")
         for category in item_categories:
             stores = category.get("stores")
             for row in stores:
+                item = DictParser.parse(row)
                 properties = {
```

Comment:
`properties` can be removed. Just do e.g. `item["postcode"] = row.get("address").get("postalCode"),` instead for simplicity?